packages/eslint-config-airbnb/CHANGELOG.md

```diff
@@ -1,126 +1,126 @@
 6.0.1 / 2016-02-21
 ==================
-- [fix] disable `newline-per-chained-call` due to an `eslint` bug (#748)
+- [fix] disable [`newline-per-chained-call`](http://eslint.org/docs/rules/newline-per-chained-call) due to an `eslint` bug [(#748)](https://github.com/airbnb/javascript/issues/748)
```

Comment:
instead of typing out the whole URL, the issue number can be referenced with just `[#748](airbnb/javascript#748)` - let's do that.

For all the eslint rules, let's use named footnotes to link to them (the rule name is ideal) rather than typing the long URLs inline (and likely duplicating them as well)